openedx_caliper_tracking/tests/local_test_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# This script file is to run the caliper app tests locally.
+
+
+FOLDER_NAME="caliper_app_testing"
+PROJECT_NAME="caliper_project"
+rm -rf $FOLDER_NAME
+mkdir $FOLDER_NAME
+cd $FOLDER_NAME
+
+printf "\n\n\n=========================================\n"
+printf "Making and Activating Virtual Environment"
+printf "\n=========================================\n"
+virtualenv venv --python=python2.7
+source venv/bin/activate
+
+printf "\n\n\n=======================\n"
+printf "Installing Dependencies"
+printf "\n=======================\n"
+pip install celery==4.3.0
```

Comment:
I think it would be better to make a requirements file and add these dependencies there.

Reply:
done.